docs/chainlink-keepers/job-scheduler.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: nodes.liquid
+section: ethereum
+date: Last Modified
+title: 'Chainlink Keepers Job Scheduler'
+whatsnext:
+  {
+    'FAQs': '/docs/chainlink-keepers/faqs/',
+  }
+---
+
+
+Learn how to register a contract with the Chainlink Keepers network and automate it using the Keepers job scheduler. Before you begin, learn how to [create a Keeper-compatible contract](../compatible-contracts).
+
+**Table of Contents**
+
++ [Register a new Upkeep](#register-a-new-upkeep)
++ [Trigger Selection](#trigger-selection)
++ [Using Time-Based Triggers](#using-time-based-triggers)
++ [Using Custom Logic Triggers](#using-custom-logic-triggers)
+
+# Register a new Upkeep
+
+To use the job scheduler, you must register a new upkeep on the Keepers network. On the landing page of the Keepers interface, click the blue **Register new Upkeep** button. 
+
+![Keepers UI Landing](/images/contract-devs/keeper/keeper-ui-landing.png)
+
+## Connecting your Wallet
+
+If you do not already have a wallet connected with the Keepers network, the interface will prompt you to do so. Click the **Connect Wallet** button and follow the remaining prompts to connect your wallet to the network.
+
+![Keepers Connect Waller](/images/contract-devs/keeper/keeper-connect-wallet.png)
+
+# Trigger Selection
+
+Once you have successfully connected your wallet, you will have options to create a trigger mechanism for automation. The trigger specifies what Keeper Nodes should look at to determine if your Upkeep should be performed. The page presents two options:
```

Comment:
Options to "select" a trigger mechanism. We won't be creating the mechanism.

Remove: "The page presents two options"

Reply:
minor nit, can be ignored

Reply:
Resolved